pkg/graph/schema.resolvers.go

```diff
@@ -1511,6 +1511,40 @@ func (r *queryResolver) UserAccount(ctx context.Context, username string) (*auth
 	return resolvers.UserAccountGetByUsername(r.store, username)
 }
 
+// SystemWorkspace is the resolver for the systemWorkspace field.
+func (r *queryResolver) SystemWorkspace(ctx context.Context, cedarSystemID string) (*model.SystemWorkspace, error) {
+	g := new(errgroup.Group)
```

Comment:
thoughts on this? i opted for this because `SystemWorkspace` will eventually be getting a ton of data from a ton of places all at once. it seemed best to start in a scalable way